retrieval/discovery/kubernetes/pod_type.go

```diff
@@ -0,0 +1,26 @@
+package kubernetes
+
+type Pod struct {
```

Comment:
I'd prefer these types in existing types.go - not much point in splitting these out to new file.